src/app/view/components/_common/action-button/action-button.html

```diff
@@ -0,0 +1,7 @@
+<p-splitButton 
+    icon="fa-plus" 
+    class="action-button" 
+    (onClick)="primaryAction()" 
+    [model]="options" 
+    [ngClass]="{'multiple-options': multipleOptions}">
```

Comment:
It's easier to read if you do:

```[class.multiple-options] = multipleOptions```

But you don't need this variable, you can do:

```[class.multiple-options] = options.length```